_posts/2022-03-12-preregistration_of_qualitative_studies.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Preregistration of qualitative studies"
+date:   2022-03-23
+author: Nandi Moksnes
+category: Practice
+tags: preregistration, qualtitiative studies
+---
+
+## Preregistration of qualitative studies
```

Comment:
```suggestion
## Preregistration of qualitative studies
```
I think this is not needed, the main header appears automatically from the post title. Correct @willu47?

Reply:
Yep, correct.